### PR TITLE
Add Notifications to AA subnav.

### PR DESCRIPTION
**Pull request by kialam**

### CODE REVIEW

File: main.yml [hunk 1]
```diff
@@ -79,11 +79,13 @@ automation-analytics:
       - /ansible
       - /ansible/automation-analytics
     sub_apps:
-      - id: ''
+      - id: clusters
```
Comment (kruai) [inline main.yml:82]: Was this change intended as well?

**Review by kruai — CHANGES_REQUESTED** [review]

File: main.yml (hunk 1 above)
Comment (kialam) [reply main.yml:82]: @kruai yes

Comment (kruai) [reply main.yml:82]: OK, just making sure. Merging now

**Review by kruai — APPROVED** [review]

File: main.yml (hunk 1 above)
Comment (kialam) [reply main.yml:82]: Thank you!

**Pull request merged**